Modules/core/Controller/CoresecureController.php

```
@@ -79,7 +79,7 @@ protected function checkRememberMeCookie() {
      */
     public function runAction($module, $action, $args = array()) {
         $modelConfig = new CoreConfig();
-        if ($modelConfig->getParam("is_maintenance") && ($this->request->getSession()->getAttribut("user_status") < 4)) {
+        if ($modelConfig->getParam("is_maintenance") && ($this->request->getSession()->getAttribut("user_status") < CoreStatus::$ADMIN)) {
                 throw new PfmException($modelConfig->getParam("maintenance_message"), 503);
```

should raise an exception not sent to sentry (PfmUserException ?)